calc-example/calc/src/ir.rs

```diff
@@ -90,19 +99,80 @@ pub struct Function {
 // ANCHOR_END: functions
 
 #[salsa::tracked]
+pub(crate) fn find_function(
```

Comment:
There is alread a tracked `find_function` in [`type_check.rs`](https://github.com/salsa-rs/salsa/blob/657b85682b5371ce7b5ef5697ef0fcd96351406d/calc-example/calc/src/type_check.rs#L26-L35), which will make the reuse of test `evaluate_example` look not working as expected ?

Reply:
Ah, *that* explains why I was seeing it in the log twice. Thanks!